### PR TITLE
feat(general): rewire submodules

### PR DESCRIPTION
**Pull request by serge1peshcoff**
1) re-wired all submodules with renamed repos
2) removed oms-core-elixir from setup, replaced it with core
3) changed default variables

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,21 +1,21 @@
-[submodule "oms-events"]
-  path = oms-events
-  url = https://github.com/AEGEE/oms-events.git
 [submodule "oms-global/docker/portal/files"]
   path = oms-global/docker/portal/files
```
Comment (WikiRik) [inline .gitmodules:2]: Can you update these as well? Not sure if global is the proper word, do you have any other words that might replace this?

Comment (serge1peshcoff) [reply .gitmodules:2]: I have no idea actually if we need it, @linuxbandit can you clarify?

Comment (serge1peshcoff) [reply .gitmodules:2]: anyway I'd rather rename/remove it in different PR, as this is not the part of 'shortening the API prefixes' epic, I didn't touch it there

**Comment by WikiRik** [comment]
Might be good to change the readme as well, most importantly the Installation section since it has links to the GitHub repo for cloning

File: .gitmodules (hunk 1 above)
Comment (linuxbandit) [reply .gitmodules:2]: indeed, postpone

**Review by linuxbandit — APPROVED** [review]

**Pull request merged**